### ISSUE TITLE
Calculate flow field directly in Fourier-Space

### ISSUE THREAD

**Issue by UnderTheCarpet** [fkjogu/stochasticsampling#34]
In GitLab by @fkjogu on Apr 12, 2017, 09:40

Instead sampling the Oseen tensor in real space and doing a convolution, use

```math
u[\psi](k) = \frac{i}{k} (1 - \hat{k} \hat{k}) \cdot \sigma[\psi] \cdot \hat{k}
```

**Comment by UnderTheCarpet** [comment]
In GitLab by @fkjogu on Apr 28, 2017, 08:38

closed

**Comment by UnderTheCarpet** [comment]
In GitLab by @fkjogu on Apr 28, 2017, 08:39

Fixed #11ab5eb9
